README.md

```diff
@@ -1,9 +1,73 @@
 # Algorithm Solutions
 
-## Repository Description
-
+Welcome to the [Algorithm Solutions](https://github.com/Alperencode/Algorithm-Solutions) repository! This is a curated collection of algorithmic challenges and their solutions, meticulously crafted to help both beginners and seasoned programmers. Whether you're preparing for a coding interview, participating in a coding challenge, or just passionate about algorithms, this repository is for you.
 This repository contains my LeetCode, CodeWars (Kata) and HackerRank problem solutions which is mostly written with Python. I'll try to keep solving problems in my free time to see different variant of questions and improve my algorithm solving skills.
 
-<br>
+Note: Files with the '-u' at the end means they are unfinished.
+
+## Table of Contents
+
+- [Features](#features)
+- [Getting Started](#getting-started)
+- [Folder Structure](#folder-structure)
+- [Contribution Guidelines](#contribution-guidelines)
+- [License](#license)
```

Comment:
This repository does not have a license so this section needs to be **removed**.